src/libraries/Microsoft.PowerFx.Connectors/Execution/HttpFunctionInvoker.cs

```diff
@@ -63,31 +65,51 @@ public HttpRequestMessage BuildRequest(FormulaValue[] args)
             // Header names are not case sensitive.
             // From RFC 2616 - "Hypertext Transfer Protocol -- HTTP/1.1", Section 4.2, "Message Headers"
             var headers = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+            string body = null;
 
             var map = _argMapper.ConvertToSwagger(args);
             foreach (var param in _argMapper._openApiParameters)
             {
-                if (map.TryGetValue(param.Name, out var paramValue))
+                if (param.In == FxParameterLocation.Body && param.Schema.Properties != null && param.Schema.Properties.Any())
```

Comment:
>if (param.In == FxParameterLocation.Body && param.Schema.Properties != null && param.Schema.Properties.Any()) [](http://example.com/codeflow?start=16&length=109)

What is this case handling? (compared to line 109)

Reply:
line 109 = raw body
here we have a description for its content

Reply:
Here we verify we have a body and check if we have a schema description for it